inc/SageHelper.php

```diff
@@ -303,4 +303,13 @@ private static function decodeStr($value)
 
         return $out;
     }
+
+    public static function str_contains($haystack, $needle)
```

Comment:
The method is nice and well thought out, but it's not following the coding convention of surrounding code by using
 * underscore_case 
 * the `false === $variable` symfony-style comparison ordering

But my main issue with this method is that the advantages of having it does not cross the threshold of usefulness to exist in the first place. 

I'm happy the `str_contains` exists in the PHP core now, but I'd never bother to implement (and then support) it and would much rather use `strpos` inline.

Reply:
done